profile/README.md

```diff
@@ -1,13 +1,9 @@
-<img src="https://github.com/w3f/Grants-Program/blob/master/src/Grants_Program.png" style="width:1300px; display: block; margin-left: auto; margin-right: auto;">
+Welcome to the Web3 Foundation Grants Program Archive.
 
-### Hi there!
+This organisation contains forks of all repositories that have ever been delivered as part of a W3F grant.
+For simplicity, the fork will usually have the same name as the original.
+In the event of a name collision, it will usually be named something like `teamName-repoName`. 
 
-This is the archive of the Web3 Foundation's Grants Program. In this organisation, you will find all repositories 
-that have ever been submitted as part of a grant. For simplicity, the repositories _usually_ have the same name 
-as the original. However, due to the occasional name collision, we might rename the fork to something like 
-`teamName-repoName`. 
+If you cannot find a project you're looking for, please [let us know](mailto:grants@web3.foundation).
```

Comment:
```suggestion
If you cannot find a project you expected here, please [let us know](mailto:grants@web3.foundation).
```

Reply:
I'm [not so sure](https://www.google.com/search?q=%22expect+you+here%22&biw=1440&bih=684&ei=3NbiYpWHO_DO7_UPtpGp8Ag&ved=0ahUKEwiV5bTYnJz5AhVw57sIHbZICo44eBDh1QMIDg&uact=5&oq=%22expect+you+here%22&gs_lcp=Cgdnd3Mtd2l6EAMyBggAEB4QBzIICAAQHhAPEAcyCAgAEB4QBxAFMgoIABAeEA8QBxAFMgYIABAeEAgyBggAEB4QCDIGCAAQHhAIMgYIABAeEAgyBggAEB4QCDIICAAQHhAIEAo6BwgAEEcQsAM6CAgAEB4QCBAHOgoIABAeEAgQBxAKSgQIQRgASgQIRhgAULoJWOgKYKsMaAJwAXgAgAFSiAHFAZIBATOYAQCgAQHIAQPAAQE&sclient=gws-wiz) about that formulation. I think [more like](https://books.google.com/ngrams/graph?content=expected+you+here%2Cexpected+to+see+you+here&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cexpected%20you%20here%3B%2Cc0%3B.t1%3B%2Cexpected%20to%20see%20you%20here%3B%2Cc0) "expected to see here", but the original seems much simpler.